language.md

```diff
@@ -22,6 +22,8 @@ Ha beszelsz magyarul, [kovess] (hungarian/malyvacukor.md)
 
 Если вы говорите по-русски, то вам [сюда](russian/zefir.md)
 
+Agar aap hindi bolte ho toh, [Hindi](Hindi/boli.md).

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.